src/librustc_trait_selection/traits/coherence.rs

```diff
@@ -435,20 +435,16 @@ fn orphan_check_trait_ref<'tcx>(
     Err(OrphanCheckErr::NonLocalInputType(non_local_spans))
 }
 
+// FIXME: Return a `Vec` without `Option` here.
 fn ty_is_non_local(tcx: TyCtxt<'tcx>, ty: Ty<'tcx>, in_crate: InCrate) -> Option<Vec<Ty<'tcx>>> {
```

Comment:
Pre-existing, but can you add a comment explaining what the return value is here?